client/src/components/Gene/Categories/CategoryResults.tsx

```diff
@@ -27,6 +28,7 @@ export const CategoryResults: React.FC = () => {
     <div className="category-results-container">
       <Tabs defaultActiveKey="1" onChange={onChange}>
         <TabPane tab="Unique Matches" key="1">
+        <TableDownloader tableName='gene_category_results' vars={{names: state.searchTerms}}/>
```

Comment:
There's an upstream issue we need to resolve to enable searching on more fields than just name (and to convert search terms to something unique, like concept ID). Once that's done, we'll need to remember to update this as well.